Common/Common.m

```diff
@@ -51,7 +51,7 @@ BOOL ValidateLoginPassword(NSString *newPassword) {
   authEnv.count = 2;
   authEnv.items = authEnvItems;
 
-  AuthorizationFlags authFlags = (kAuthorizationFlagExtendRights);
+  AuthorizationFlags authFlags = (kAuthorizationFlagExtendRights | kAuthorizationFlagDestroyRights);
```

Comment:
We just want to check if the password is correct, we don't actually want the associated rights.